### PR TITLE
Set default for FETCHMAIL_ENABLED

### PR DESCRIPTION
**Pull request by ghostwheel42**
## What type of PR?

bug-fix

## What does this PR do?

Set the default for FETCHMAIL_ENABLED to true in the admin container.
This keeps existing functionality for people upgrading without re-creating the `mailu.env`.

### CODE REVIEW

File: tests/compose/fetchmail/mailu.env [hunk 1]
```diff
@@ -83,6 +83,9 @@ RELAYNETS=
 # Will relay all outgoing mails if configured
 RELAYHOST=
 
+# Show fetchmail functionality in admin interface
+FETCHMAIL_ENABLED=false
```
Comment (Diman0) [inline tests/compose/fetchmail/mailu.env:87]: Should fetchmail not be enabled in mailu.env (tests/compose/fetchmail/mailu.env)  for the fetchmail test?
Maybe the fetchmail_delay could also be lowered to 15 seconds. 
I know the fetchmail test only starts all containers, but this at least tests if the fetchmail container can start fine and connect to admin to retrieve the fetchmail pull settings.

Comment (ghostwheel42) [reply tests/compose/fetchmail/mailu.env:87]: Hmm. Don't know - where is the test code?
I'll change it. Maybe we should update all test cases to the new files created by the setup utility before the next release.

**Review by Diman0 — APPROVED** [review]
Lgtm

**Comment by ghostwheel42** [comment]
bors retry

**Pull request closed without merging**